test/dom/equivalence.ts

```diff
@@ -347,4 +347,13 @@ describe("Equivalence", () => {
       )
     );
   });
+  it("equals() for Struct with duplicate fields", () => {
+    let value1: Value = load("{ foo: 'bar', foo: 'baz' }")!;
+    let value2: Value = load("{ foo: 'bar', foo: 'baz' }")!;
+    let value3: Value = load("{ foo: 'bar', foo: 'qux' }")!;
```

Comment:
Can you add a test that verifies that field order is not considered for equality? e.g.

```ts
{foo: 1, baz: true, foo: 2}
// is equal to
{foo: 2, foo: 1, baz: true}
```